Classes/Issues/Labeled/IssueLabeledCell.swift

```diff
@@ -9,95 +9,78 @@
         ]
-        let actor = NSAttributedString(string: model.actor, attributes: actorAttributes)
-
+        
+        let attributedString = NSMutableAttributedString(string: model.actor, attributes: actorAttributes)
+        
+        // Action
+        
+        let actionAttributes = [
+            NSAttributedStringKey.foregroundColor: Styles.Colors.Gray.medium.color,
+            NSAttributedStringKey.font: Styles.Fonts.secondary
+        ]
+        
         let actionString: String
+        
         switch model.type {
-        case .added: actionString = NSLocalizedString(" added", comment: "")
-        case .removed: actionString = NSLocalizedString(" removed", comment: "")
+        case .added: actionString = NSLocalizedString(" added  ", comment: "")
+        case .removed: actionString = NSLocalizedString(" removed  ", comment: "")
         }
-        let actionAttributes = [
+        
+        attributedString.append(NSAttributedString(string: actionString, attributes: actionAttributes))
+        
+        // Label
+        
+        let labelColor = model.color.color
+        
+        let labelAttributes: [NSAttributedStringKey: Any] = [
+            NSAttributedStringKey.font: Styles.Fonts.secondaryBold,
+            NSAttributedStringKey.backgroundColor: labelColor,
+            NSAttributedStringKey.foregroundColor: labelColor.textOverlayColor ?? .black,
+            MarkdownAttribute.label: model.title
+        ]
+        
+        attributedString.append(NSAttributedString(string: model.title, attributes: labelAttributes))
+        
+        // Date
+        
+        let dateAttributes: [NSAttributedStringKey: Any] = [
+            NSAttributedStringKey.font: Styles.Fonts.secondary,
             NSAttributedStringKey.foregroundColor: Styles.Colors.Gray.medium.color,
-            NSAttributedStringKey.font: Styles.Fonts.secondary
+            MarkdownAttribute.details: DateDetailsFormatter().string(from: model.date)
         ]
-        let action = NSAttributedString(string: actionString, attributes: actionAttributes)
-        let descriptionText = NSMutableAttributedString(attributedString: actor)
-        descriptionText.append(action)
-        descriptionButton.setAttributedTitle(descriptionText, for: .normal)
-
-        let color = UIColor.fromHex(model.color)
-        labelButton.backgroundColor = color
-
-        labelButton.setTitle(model.title, for: .normal)
-        labelButton.setTitleColor(color.textOverlayColor, for: .normal)
-
-        dateLabel.setText(date: model.date)
+        
+        attributedString.append(NSAttributedString(string: "  \(model.date.agoString)", attributes: dateAttributes))
+        
+        // Set
+        
+        let centerVertical = (Styles.Sizes.labelEventHeight - Styles.Fonts.secondaryBold.lineHeight) / 2
+        let inset = UIEdgeInsets(
+            top: centerVertical,
+            left: Styles.Sizes.eventGutter,
+            bottom: centerVertical,
```

Comment:
When we move string generation into the model, let's just use insets for the top/bottom so the cell can be sized according to the string size:

```swift
UIEdgeInsets(
  top: Styles.Sizes.inlineSpacing,
  left: Styles.Sizes.eventGutter,
  bottom: Styles.Sizes.inlineSpacing,
  right: Styles.Sizes.eventGutter
)
```

(we can move this inset to a const since it'll be used so much)

Reply:
The issue is the top/bottom has to be 7 otherwise the spacing is different to the other events. I was testing with the issue attached so if the sizing wasn't the same then you could visible see different padding (and my need for consistency was hurting).

I'll work out something else